adrienlbt/hexagonal-maker-bundle/1.2/config/packages/hexagonal_maker_bundle.yaml

```
@@ -0,0 +1,4 @@
+hexagonal_maker:
+    application_path: 'Application'
+    domain_path: 'Domain'
+    infrastructure_path: 'Infrastructure'
```

Since your bundle is only available in `dev` env, this configuration will fail in `prod` and needs to be wrapped in `when@dev`.

```suggestion
when@dev:
    hexagonal_maker:
        application_path: 'Application'
        domain_path: 'Domain'
        infrastructure_path: 'Infrastructure'
```